openstack/tools/charmed_openstack_functest_runner.sh

```diff
@@ -0,0 +1,64 @@
+#!/bin/bash -ux
+source ~/novarc
+CIDR=`openstack subnet show subnet_${OS_USERNAME}-psd-extra -c cidr -f value`
+FIP_MAX=$(ipcalc $CIDR| awk '$1=="HostMax:" {print $2}')
+FIP_MIN=$(ipcalc $CIDR| awk '$1=="HostMin:" {print $2}')
+FIP_MIN_ABC=${FIP_MIN%.*}
+FIP_MIN_D=${FIP_MIN##*.}
+FIP_MIN=${FIP_MIN_ABC}.$(($FIP_MIN_D + 64))
```

Comment:
This turns out to be gateway_ip + 64 since it's always .1 by default, and as long as this is a /25 or larger it's fine, smaller and it will be greater than the max. It's only until they get the gate fixed anyways...